test/unit/Dredd-test.js

```diff
@@ -91,7 +90,7 @@ describe('Dredd class', () => {
         configuration = {
           server: 'http://127.0.0.1:3000/',
           options: {
-            silent: true,
```

Comment:
NIT, it would be easier to review if the commit that removed the option also removed it from the code. It's harder to review when things like level is removed but a test using level was updated in separate commit etc. Mostly feedback for consideration in future PRs I don't expecyou to change anything at this point.

Not to mention this separation would make it problematic for anyone to use git-bisect on this project as tests may not pass on some commits.

Reply:
That's a good point.